src/components/EmojiPicker/EmojiPicker.tsx

```diff
@@ -45,34 +48,39 @@ const EmojiPicker = forwardRef((props, ref) => {
      *
      * Don't directly get the ref from emojiPopoverAnchorRef, instead use getEmojiPopoverAnchor()
      */
-    const getEmojiPopoverAnchor = useCallback(() => emojiPopoverAnchorRef.current || emojiPopoverAnchorRef, []);
+    const getEmojiPopoverAnchor = useCallback(() => emojiPopoverAnchorRef.current ?? emojiPopoverAnchorRef, []);
```

Comment:
I think there is a bug in the code, we are lacking current here
```suggestion
    const getEmojiPopoverAnchor = useCallback(() => emojiPopoverAnchorRef.current ?? emojiPopoverAnchorRef?.current, []);
```